mozilla_vpn/explores/subscriptions.explore.lkml

```diff
@@ -90,10 +87,8 @@ explore: +subscriptions {
       ]
     }
 
-    # Please specify a datagroup_trigger or sql_trigger_value
-    # See https://looker.com/docs/r/lookml/types/aggregate_table/materialization
     materialization: {
-      sql_trigger_value: SELECT CURRENT_DATE();;
+      sql_trigger_value:  v
```

Comment:
this doesn't seem right

Reply:
Thanks for catching this.  It was a typo.  I just fixed it.